graylog2-server/src/main/java/org/graylog/plugins/sidecar/migrations/V20180212165000_AddDefaultCollectors.java

```diff
@@ -328,48 +470,93 @@ private void removeConfigPath() {
+                                                String nodeOperatingSystem,
+                                                String executablePath,
+                                                String executeParameters,
+                                                String validationCommand,
+                                                String defaultTemplate) {
         Collector collector = null;
         try {
             collector = collectorService.findByNameAndOs(collectorName, nodeOperatingSystem);
             if (collector == null) {
-                final String msg = "Couldn't find collector '{} on {}' fixing it.";
-                LOG.error(msg, collectorName, nodeOperatingSystem);
+                LOG.error("Couldn't find collector '{} on {}' fixing it.", collectorName, nodeOperatingSystem);
                 throw new IllegalArgumentException();
             }
         } catch (IllegalArgumentException ignored) {
             LOG.info("{} collector on {} is missing, adding it.", collectorName, nodeOperatingSystem);
-            final Collector newCollector;
-            newCollector = Collector.create(
-                    null,
-                    collectorName,
-                    serviceType,
-                    nodeOperatingSystem,
-                    executablePath,
-                    executeParameters,
-                    validationCommand,
-                    defaultTemplate
-            );
             try {
-                return collectorService.save(newCollector).id();
+                return Optional.of(collectorService.save(Collector.create(
+                        null,
+                        collectorName,
+                        serviceType,
+                        nodeOperatingSystem,
+                        executablePath,
+                        executeParameters,
+                        validationCommand,
+                        defaultTemplate
+                )));
             } catch (Exception e) {
-                LOG.error("Can't save collector " + collectorName + ", please restart Graylog to fix this.", e);
+                LOG.error("Can't save collector '{}'!", collectorName, e);
             }
         }
 
         if (collector == null) {
-            LOG.error("Unable to access fixed " + collectorName + " collector, please restart Graylog to fix this.");
-            return null;
+            LOG.error("Unable to access fixed '{}' collector!", collectorName);
+            return Optional.empty();
         }
 
-        return collector.id();
+        return Optional.of(collector);
+    }
+
+    private void ensureConfigurationVariable(String name, String description, String content) {
+        ConfigurationVariable variable = null;
+        try {
+            variable = configurationVariableService.findByName(name);
+            if (variable == null) {
+                LOG.error("Couldn't find sidecar configuration variable '{}' fixing it.", name);
```

Comment:
```suggestion
                LOG.info("Couldn't find sidecar configuration variable '{}' fixing it.", name);
```
This is not an error